clients/src/main/java/org/apache/kafka/clients/Metadata.java

```diff
@@ -217,12 +217,12 @@ public synchronized boolean updateRequested() {
         }
     }
 
-    public synchronized Map<String, Uuid> topicIds() {
-        return cache.topicIds();
+    public synchronized Uuid topicId(String topicName) {
```

Comment:
In the 3.0 patch, I think we added javadoc comments: https://github.com/apache/kafka/pull/10952/files#diff-97c2911e6e1b97ed9b3c4e76531a321d8ea1fc6aa2c727c27b0a5e0ced893a2cR221.